x-pack/plugins/monitoring/public/components/cluster/overview/logstash_panel.js

```diff
@@ -139,15 +139,14 @@ function LogstashPanelUi(props) {
                   { pipelineCount: props.pipeline_count }
                   )}
                 >
-                  <Tooltip
-                    text={props.intl.formatMessage({
+                  <EuiToolTip
+                    content={props.intl.formatMessage({
                       id: 'xpack.monitoring.cluster.overview.logstashPanel.betaFeatureTooltip',
                       defaultMessage: 'Beta Feature' })}
-                    placement="bottom"
-                    trigger="hover"
+                    position="bottom"
                   >
                     <span className="kuiIcon fa-flask betaIcon" />
```

Comment:
`kui` -> Could switch this to the beaker icon and use EUI. Not sure about the `betaIcon` CSS though (is this feature reasonably still in beta anyway?). Feel free to bump into a separate issue/PR.